.github/workflows/demo.yml

```diff
@@ -1,8 +1,10 @@
-name: Deploy modules to dev
+name: Deploy modules to demo project
 on:
   push:
     branches:
       - main
+env:
+  PYTHON_VERSION: '3.11'
```

Comment:
Same argument here, if we allow `3.9` as the lower bound we should use it to ensure that we do not use Python syntax for 3.10 and above